docs/DEVELOPMENT.md

```diff
@@ -2,7 +2,8 @@
 
 ## Requirements
 
-* Python 3
+* Python 3.10.0
```

Comment:
This now conflicts with the note, below.

Reply:
Good point, I'd meant to delete that 🤦 It seems like whatever the issue was has now been resolved, so I'll delete.